app/helpers/application_helper/toolbar/miq_template_center.rb

```diff
@@ -71,7 +71,8 @@ class ApplicationHelper::Toolbar::MiqTemplateCenter < ApplicationHelper::Toolbar
           'fa fa-search fa-lg',
           N_('Check Compliance of the last known configuration for this Template'),
           N_('Check Compliance of Last Known Configuration'),
-          :confirm => N_("Initiate Check Compliance of the last known configuration for this Template?")),
+          :confirm => N_("Initiate Check Compliance of the last known configuration for this Template?"),
+          :klass   => ApplicationHelper::Button::MiqCheckCompliance),
```

Comment:
@vecerek I wonder if this toolbar is actually used anywhere, I can not find it anywhere.

@h-kataria any idea, where this could be?

I just know, that `miq_templates_center` is in _Services -> Workloads -> Templates & Images accordion -> select Template or Image_. I'd expect `miq_template_center` to appear after selecting one specific template, but I'm getting `x_template_cloud_center` instead.